### ISSUE TITLE
Cluster accuracy

### ISSUE THREAD

**Issue by pp1016** [wvangansbeke/Unsupervised-Classification#50]
Could you explain how the cluster accuracy (ACC) calculated? It is also not well defined in paper, thanks.

**Comment by wvangansbeke** [comment]
Hi @pp1016,

It's the accuracy metric = number of correctly classified samples/ total amount of samples in the validation set. Nothing fancy, just the same as in supervised learning.

**Comment by pp1016** [comment]
> Hi @pp1016,
> 
> It's the accuracy metric = number of correctly classified samples/ total amount of samples in the validation set. Nothing fancy, just the same as in supervised learning.

Thanks for your reply but I am still a little confused. I saw you also use NMI metric for comparison, that make sense as in unsupervised learning, the evaluation result is just the clusters with no actual labels assigned on each image/data. In many other papers, they supervised train a classifier on top of the learned feature to report the classification accuracy, is that the same for you? As I saw this term "cluster accuracy" in another paper and they define it as The accuracy obtained when assigning each cluster to its most represented class), thanks!

**Comment by wvangansbeke** [comment]
We use the Hungarian matching algorithm, to match the predicted clusters with the ground truth classes. Now, you can calculate the accuracy as usual. Prior work also adopts this matching algorithm. I refer you to the code for the exact implementation.

**Comment by pp1016** [comment]
> We use the Hungarian matching algorithm, to match the predicted clusters with the ground truth classes. Now, you can calculate the accuracy as usual. Prior work also adopts this matching algorithm. I refer you to the code for the exact implementation.

Thanks!
